src/cases/store/reducers/index.ts

```diff
@@ -2,15 +2,18 @@ import {ActionReducerMap, createFeatureSelector} from '@ngrx/store';
 
 import * as fromCaseCreate from './create-case.reducer';
 import * as fromCaseSearchFilters from './search-filter.reducer';
+import * as fromCaseListFilters from './case-list-filter.reducer';
```

Comment:
remove the Filters from both Search and List

Reply:
Done, thanks.